### PR TITLE
Feature/get navigation state

### PR DESCRIPTION
**Pull request by emiax**

### CODE REVIEW

**Comment by emiax** [comment]
@alexanderbock Let me know if you think this is ready for merge

File: src/interaction/navigationhandler.cpp [hunk 1]
```diff
@@ -568,21 +572,20 @@ scripting::LuaLibrary NavigationHandler::luaLibrary() {
...
                 &luascriptfunctions::getNavigationState,
                 {},
-                "[string]",
-                "Return the current navigation state as a lua table. "
-                "The optional argument is the scene graph node to "
-                "use as reference frame. By default, the reference frame will picked "
-                "based on whether the orbital navigator is currently following the "
-                "anchor node rotation. If it is, the anchor will be chosen as reference "
-                "frame. If not, the reference frame will be set to the scene graph root."
+                "[table]",
```
Comment (emiax) [inline src/interaction/navigationhandler.cpp]: This is the optional input argument, and not the return value, right? It should be [string].

Comment (alexanderbock) [reply src/interaction/navigationhandler.cpp]: Yep.. you are right.. oopsie

Comment (emiax) [reply src/interaction/navigationhandler.cpp]: Thanks for reverting

**Pull request merged**